server/src/routes/api/v1/shoeReviewsRouter.js

```diff
@@ -0,0 +1,24 @@
+import express from "express"
+import { ValidationError } from "objection"
+import { Review } from "../../../models/index.js"
+import cleanUserInput from "../../../../services/cleanUserInput.js"
+
+const shoeReviewsRouter = new express.Router({ mergeParams: true })
+
+shoeReviewsRouter.post("/", async (req, res) => {
+  const bodyInput = cleanUserInput(req.body)
+  const { score, body, userId } = bodyInput
+  const shoeId = parseInt(req.params.shoeId)
+
+  try {
+    const newReview = await Review.query().insertAndFetch({ score: score, body: body, shoeId:shoeId, userId: parseInt(userId) })
```

Comment:
should not have to parse the id (even if it's a string postgres will type convert the value for us from string to integer when it saves to the database)

look back at the authentication articles for how we should be accessing the currently signed in user on the backend here